pkgs/development/tools/rgp/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, stdenv
+, makeWrapper
+, fetchurl
+, autoPatchelfHook
+, dbus
+, fontconfig
+, freetype
+, glib
+, libglvnd
+, libX11
+, libxcb
+, libXi
+, ncurses5
+, qtbase
+, zlib
+}:
+with lib;
```

Comment:
I just now noticed that this `with` isn't necessary at all. Given the previously linked IRC discussion, it should probably be removed. I don't think its important enough to hold off merge even longer though.